roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-operator.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cluster-monitoring-operator
+  namespace: openshift-monitoring
+  labels:
+    k8s-app: cluster-monitoring-operator
+    managed-by-channel-operator: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: cluster-monitoring-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: cluster-monitoring-operator
+        tectonic-app-version-name: tectonic-monitoring
+    spec:
+      containers:
+      - image: quay.io/coreos/cluster-monitoring-operator:198d25d6
+        name: cluster-monitoring-operator
+        args:
+        - "-namespace=openshift-monitoring"
+        - "-configmap=cluster-monitoring-config"
+        - "-logtostderr=true"
+        - "-v=4"
+        resources:
+          limits:
+            cpu: 20m
+            memory: 50Mi
+          requests:
+            cpu: 20m
+            memory: 50Mi
+      restartPolicy: Always
+      securityContext:
```

Comment:
This key needs to go away

Reply:
I thought we wanted the Operator to run as non-root?

Reply:
We do. OpenShift will automatically generate a securityContext (including runAsNonRoot), and setting it explicitly here will break stuff (this is the reason we have the various admin role hacks).

We should remove all explicit `securityContext` keys from here and TPO/PO manifests and let the platform (origin/k8s) do defaulting. If our pods aren't landing in the "restricted" (non-root, random uid) context by default, that's something we can fix at the platform config level.